package.json

```diff
@@ -44,11 +44,11 @@
     "handlebars": "^4.7.2",
     "handlebars-loader": "^1.7.1",
     "html-webpack-plugin": "^4.0.0-beta.8",
-    "i18next": "^19.0.0",
+    "i18next": "^20.2.1",
     "jquery": "^3.4.1",
     "jquery-caret-plugin": "^1.5.2",
     "jquery-validation": "^1.19.0",
-    "katex": "^0.12.0",
+    "katex": "^0.13.2",
```

Comment:
Can you do a bit of manual testing of the KaTeX upgrade before merging?

I think it should be fine, but it's a sufficiently complex dependency that it's worth a test.

Reply:
I sent myself, like, $$\sqrt 5$$ and the display equivalent. Is there something more complicated you want to test?

Reply:
Verified some more complicated things including the scrollbar thing from #14422.